README.md

```diff
@@ -44,45 +70,112 @@ The structure should appear like so:
 \---specification
 |    +---automation
 |    |   \---resource-manager
-|    |       \---Microsoft.Automation
-|    |           \---stable
-|    |               \---2015-10-31
-|    |                   \---examples
+|    |       +---Microsoft.Automation
+|    |       |   \---stable
```

Comment:
Here in the specific case of Compute, where the service included 4 different RPs, we will create folders for different RPs here